tests/functional/eyes_selenium/selenium/test_specific_cases.py

```diff
@@ -59,3 +61,22 @@ def test_abort_eyes(eyes, driver):
     eyes.open(driver, "Python VisualGrid", "TestAbortSeleniumEyes")
     eyes.check_window()
     eyes.abort()
+
+
+def test_ie_viewport_screenshot(eyes, webdriver_module):
+    sauce_url = "https://{username}:{password}@ondemand.saucelabs.com:443/wd/hub".format(
+        username=os.getenv("SAUCE_USERNAME", None),
+        password=os.getenv("SAUCE_ACCESS_KEY", None),
+    )
+    driver = webdriver_module.Remote(
+        command_executor=sauce_url,
+        desired_capabilities={
+            "browserName": "internet explorer",
+            "platform": "Windows 10",
+            "version": "11.285",
+        },
+    )
+    driver.get("http://applitools.github.io/demo/TestPages/FramesTestPage")
+    eyes.open(driver, "Python SDK", "TestIEViewportScreenshot")
+    eyes.check_window()
```

Comment:
What the stitch mode? Where do you check full page?
We need to run both viewport and full page on both CSS and Scroll stitch modes (4 permutations total).

Reply:
Default stitch mode is Scroll. I'll add CSS stitch ode and FPS here